_posts/en/newsletters/2023-11-08-newsletter.md

```diff
@@ -0,0 +1,202 @@
+---
+title: 'Bitcoin Optech Newsletter #276'
+permalink: /en/newsletters/2023/11/08/
+name: 2023-11-08-newsletter
+slug: 2023-11-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Discontinuation of mailing lists:** administrators for the
+  Bitcoin-Dev mailing list [announced][togami lists] in the
+  #bitcoin-core-dev IRC channel that the organization hosting the
+  mailing lists plans to cease hosting any mailing lists after the end
+  of the year.  We assume that this also affects the Lightning-Dev
+  mailing list, which is hosted by the same organization.
+
+    In the months prior to the announcement, some well-established
+    developers had begun experimenting with discussions on the
+    [DelvingBitcoin][] web forum.  Optech will begin monitoring that
+    forum for interesting or important discussions effective
+    immediately.  We've also added a [document][sources] to our site
+    that lists all the discussion groups we track for news, and we'll
+    keep that list up to date if we determine that the primary site for
+    online discussion has moved elsewhere.
+
+- **HTLC aggregation with covenants:** Johan Torås Halseth
+  [posted][halseth agg] to the Lightning-Dev mailing list a suggestion
+  for using a [covenant][topic covenants] to aggregate multiple
+  [HTLCs][topic htlc] into a single output that could be spent all at
+  once if a party knew all the preimages.  If a party only knew some of
+  the preimages, they could claim just those and then the remaining
+  balance could be refunded to the other party.  Halseth notes that this
+  would be more efficient onchain and could make it more difficult to
+  perfrom certain types of [channel jamming attacks][topic channel
+  jamming attacks].
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting, highlighting some of the important questions and
+answers.  Click on a question below to see a summary of the answer from
+the meeting.*
+
+[Fee Estimator updates from Validation Interface/CScheduler thread][review club 28368]
+is a PR by Abubakar Sadiq Ismail (ismaelsadeeq)
+that modifies the way the transaction fee estimator integrates with the
+rest of the `bitcoind` client.
+It moves fee estimator updates from occurring synchronously during mempool
+updates (transactions being added or removed) to instead occur asynchronously.
+This is a performance improvement.
+
+When a new block is found, its transactions
+to be removed from the mempool.
+Since block validation and relay are performance-critical, it's beneficial
+to reduce the required amount of work -- such as updating the fee estimator.
+{% include functions/details-list.md
```

Comment:
details-list.md can handle 10 Q&A's as an upper bound. I think its probably best either way to trim the number of questions highlighted this week.

IMHO q3,q5,q6 could be candidates for trimming, but Im not deeply familiar with the PR and review

Reply:
I agree q3 and q5 are good candidates for trimming. Would also suggest q11.